FWCore/ParameterSet/bin/edmWriteConfigs.cpp

```diff
@@ -136,18 +138,34 @@ namespace {
     }
   }
 
+  bool open_temp(std::string& path, std::ofstream& f) {
```

Comment:
```suggestion
  bool open_temp(std::string const& path, std::ofstream& f) {
```
or
```suggestion
  bool open_temp(std::string path, std::ofstream& f) {
```

Reply:
Can't pass as a `const`, since `mkstemp` modifies string passed to it